wire-ios-sync-engine/Source/Synchronization/StrategyDirectory.swift

```diff
@@ -413,6 +354,84 @@ public class StrategyDirectory: NSObject, StrategyDirectoryProtocol {
         ]
     }
 
+    func makeRemainingStategies(
```

Comment:
This is not very elegant but I would prefer to improve it in a separate PR.

Reply:
suggestion: add a comment to which kind of strategies we're talking about, maybe:
```suggestion
    func makeClientRelatedStrategies(
```